### DISCUSSION TITLE
v1.4-alpha.4

### DISCUSSION THREAD

**Discussion by arekf** [yattee/yattee#132] [Announcements]
## What's Changed
* Added Picture in Picture for iOS
* Added playback rate option
* Added 4K, 60fps resolution (#118)
* Added 8K resolutions
* Added resolutions for 50fps and 48fps (#120)
* Format VP9/WEBM is now preferred for MPV (#128)
* Fixed displaying watch history of videos in the queue
* Fixed behavior of restoring last played video
* Fixed disabled state of play/pause buttons
* Improved player drawing performance on some devices
* Includes changes from stable 1.3.3 release:
  * User playlists support for Piped accounts
  * Channel subscriber count is retrieved without additional query with Piped
* Other minor changes and fixes

**Full Changelog**: https://github.com/yattee/yattee/compare/v1.4-alpha.3...v1.4-alpha.4

<hr /><em>This discussion was created from the release <a href='https://github.com/yattee/yattee/releases/tag/v1.4-alpha.4'>v1.4-alpha.4</a>.</em>

**Comment by bachig26** [comment]
Hi,

These are the two issues that I see immediately when tried 1.4-alpha4, you might have known this.
and I didn't use much 1.4 alpha3 after looking for the difference on the release day, so i forgot how it was. so maybe this issue is still from there.

1. when a video enters fullscreen, the orientation is locked to a default landscape direction and I wanted to flip it to other landscape but doesn't work.
2. during one of my video playback, the video just stopped and only audio is running. but the returns to the next/ audio synced location when I closed the fullscreen.

I just wanted to let you know of my initial issue of this release.

_ps. when pip is used, the video jumped from default 1080p mp4 to hls [as it is avplayer for pip], i wanted to know if the resolution or quality drop will occur during playback as you may know that quality sometimes drops in avplayer for yattee?_

**Reply by bachig26** [reply]
Newly found bug after installing through TestFlight:
- Framarate drops throughout video playback
- As I mentioned in previous message about the fullscreen and Orientation issue with MPV, the **Orientation options in Player Settings** is completely useless now.

Improvement needed:
- you may have this in your todo list, the subtitle selector in the MPV player is missing.
